src/components/test-slot/test-slot/test-slot.ts

```diff
@@ -128,4 +129,13 @@ export class TestSlotComponent implements SlotComponent, OnInit {
     const daysToAdd = today.isoWeekday() === 5 ? 3 : today.isoWeekday() === 6 ? 2 : 1;
     return  moment().add(daysToAdd, 'days').startOf('day').toDate();
   }
+
+  getExaminerId(): number {
+    let returnValue = null;
+    if (this.delegatedTest) {
```

Comment:
already done a 'this.delegatedTest' in the if, so no need for the elvis operator check. Should just return slot.examinerId

Reply:
fair point 👍